activities/communication/communication-remote-first.md

```diff
@@ -0,0 +1,29 @@
+---
+type: Resource
+---
+
```

Comment:
Add a top level heading (with one #) with the title of the page.

Reply:
Done!